yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -155,10 +155,10 @@ private[spark] class Client(
 
       // Get a new application from our RM
       val newApp = yarnClient.createApplication()
-      val newAppResponse = newApp.getNewApplicationResponse()
-      appId = newAppResponse.getApplicationId()
+      val newAppResponse = newApp.getNewApplicationResponse
```

Comment:
These are Java methods? I think it's canonical to call them with empty args. Not worth changing anyway